src/s2e/Plugins/Lua/LuaS2EExecutionState.cpp

```diff
@@ -124,5 +127,19 @@ int LuaS2EExecutionState::debug(lua_State *L) {
 
     return 0;
 }
+
+int LuaS2EExecutionState::registerInstructionAnnotation(lua_State *L){
```

Comment:
This must go to LuaInstructionAnnotation plugin instead. LuaS2EExecutionState must not depend on any S2E plugins.
You will need to build a mechanism to call arbitrary plugins from Lua code, which will probably go into LuaS2E.cpp and will mimick the S2E::getPlugin() call.